docs/docs/reaching-out-to-user.mdx

```diff
@@ -0,0 +1,459 @@
+---
+id: reaching-out-to-user
+sidebar_label: Reaching Out to the User
+title: Reaching Out to the User
+---
+
+Sometimes you want your assistant to reach out to the user without the user's prompting.
+For example, you might want the assistant to send a message when the user opens the chat window,
+or you might want to prompt the user if they haven't sent a message for a while.
+This page is a guide to enabling your assistant to reach out to the user proactively.
+
+
+## Reaching out first
+
+To have your assistant reach out to the user at the beginning of a conversation,
+configure your [messaging or voice channel](messaging-and-voice-channels.mdx) to send a payload to the bot that
+will trigger the appropriate response. For example, in Sara, the Rasa bot, when a
+user opens the chat window, the chat widget sends the payload `/greet`, so that
+Sara greets the user with a welcome message. How you send the initial payload
+will depend on the channel(s) you are using.
```

Comment:
hm, this paragraph is kind of confusing - should we clarify that some channels have this functionality inbuilt by default, others do not?